google-play-games/ios/README.md

```diff
@@ -119,5 +131,397 @@ project(":ios") {
+    </array>
+```
+## Coding
+
+### Action Resolver
+
+LibGDX is Crossplatform but to interact with native libraries you need to write system specific methods.
+i.e: You don't call the Google Games Services the same way on Android than on iOS.
+
+To do that, first we create an interface that list the platform specific method.
+This class is in our core project
+In our case we will only implement the `SignIn` method, although that can easily be extended.
+
+#### interface 
+
+Create `ActionResolver.java` in your `core` project
+
+```
+package com.mygdx.game;
+
+
+public interface ActionResolver {
+
+    public void signIn();
+}
+```
+
+##### ActionResolver Desktop
+
+No Google Games Services implementation for Dektop. Calling SignIn when running on Desktop won't do anythning.
+
+Create `ActionResolverDesktop.java` in your `desktop` project
+
+```
+package com.mygdx.game.desktop;
+
+import com.mygdx.game.ActionResolver;
+
+public class ActionResolverDesktop implements ActionResolver {
+
+
+    @Override
+    public void signIn() {
+		// nothing happens.
+    }
+}
+```
+
+Your desktop Launcher is modified as follow to call the dummy desktop Action Resolver
+
+```
+package com.mygdx.game.desktop;
+
+import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
+import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
+import com.mygdx.game.MyGdxGame;
+
+public class DesktopLauncher {
+
+
+	public static void main (String[] arg) {
+		ActionResolverDesktop actionResolverDesktop = new ActionResolverDesktop();
+		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
+		new LwjglApplication(new MyGdxGame(actionResolverDesktop), config);
+	}
+	
+}
+```
+
+##### ActionResolver Android
+
+No Google Games Services implementation for Android on this tutorial at this point. Calling SignIn when running on Android won't do anythning.
+
+Create `ActionResolverAndroid.java` in your `android` project
+
+```
+package com.mygdx.game;
+
+public class ActionResolverAndroid implements ActionResolver{
+
+    @Override
+    public void signIn() {
+
+    }
+}
+```
+
+Your Android Launcher is modified as follow to call the dummy Android Action Resolver
+
+```
+package com.mygdx.game;
+
+import android.os.Bundle;
+
+import com.badlogic.gdx.backends.android.AndroidApplication;
+import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
+
+public class AndroidLauncher extends AndroidApplication {
+
+	private ActionResolverAndroid actionResolverAndroid;
+
+	@Override
+	protected void onCreate (Bundle savedInstanceState) {
+		actionResolverAndroid = new ActionResolverAndroid();
+		super.onCreate(savedInstanceState);
+		AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
+		initialize(new MyGdxGame(actionResolverAndroid), config);
+	}
+
+}
+
+```
+
+
+##### ActionResolverHTML
+
+No Google Games Services implementation for HTML on this tutorial. Calling SignIn when running on Android won't do anythning.
```

Comment:
Should be "Calling SignIn when running on **HTML** won't do anything" 
Typo, `anythning`